service-python/main_consumer.py

```diff
@@ -37,24 +39,33 @@ def _create_connection(self):
             except Exception:
                 logging.warning(f"RabbitMQ Connection Failed. Retrying in 30s ({i + 1}/{self.config['retry_limit']})")
```

Comment:
One more thing. Can we add information about the exception here? Looks like logging.warning has a exc_info parameter where you can specify the exception instance.